src/test/regress/expected/failure_connection_establishment.out

```diff
@@ -225,8 +225,8 @@ SELECT count(*) FROM single_replicatated WHERE key = 100;
 
 RESET client_min_messages;
 -- verify get_global_active_transactions works when a timeout happens on a connection
-SELECT get_global_active_transactions();
- get_global_active_transactions
+SELECT * FROM get_global_active_transactions() WHERE datid != 0;
```

Comment:
as we discussed, lets use another field than `datid`

Reply:
And, for reference, we have to add these because now maintenance daemon processes shows up in `get_global_active_transactions/get_all_active_transactions`. 

We don't mind `get_global_active_transactions/get_all_active_transactions`.  But, they also current show up in citus_dist/worker_stat_activity views, where we'd like to avoid maintenance processes. That is going to happen with another PR